media/css/base/mozilla-modal.less

```diff
@@ -28,52 +28,63 @@ html.noscroll body {
 
         .inner {
             position: relative;
-            @shadow: 0 1px 22px rgba(0, 0, 0, 0.8);
-            .box-shadow(@shadow);
 
-            header {
+            &> header {
                 display: block;
-                background: #0095dd;
-                padding: 10px @baseLine;
-                height: 38px;
-                .font-size(26px);
+                height: auto;
+                .font-size(48px);
+                line-height: 1.3;
+                padding: 0 (@gridGutterWidth * 3);
+                text-align: center;
                 color: #fff;
                 .open-sans-light;
-                .transition;
             }
 
             #modal-close {
                 z-index: 99;
                 position: absolute;
-                top: 9px;
+                top: -20px;
                 right: 9px;
                 cursor: pointer;
 
                 a {
-                    color: #fff;
-                    vertical-align: middle;
-                    padding: 0 3px;
+                    // TODO: remove <a> element from mozilla-modal.js with mozID rollout
+                    display: none;
                 }
 
                 .button {
+                    background-color: transparent;
+                    // png fallback for IE8
+                    background-image: url(/media/img/mozid/modal-close.png);
+                    // svg for good browsers
+                    background-image: none, url(/media/img/mozid/modal-close.svg);
+                    background-position: center center;
+                    background-repeat: no-repeat;
+                    background-size: 22px 22px;
+                    filter: progid:DXImageTransform.Microsoft.gradient(enabled = false);
+                    -ms-filter: progid:DXImageTransform.Microsoft.gradient(enabled = false);
                     padding: 0;
-                    min-width: 34px;
-                    .font-size(30px);
-                    vertical-align: middle;
-                }
-
-                &:hover {
-                    a {
-                        text-decoration: underline;
+                    min-width: 0;
+                    width: 42px;
+                    height: 42px;
+                    .image-replaced();
+                    border-radius: 50%;
+                    border: 2px solid #fff;
+                    .transition(all 0.2s ease);
```

Comment:
Not related to this change, but this `transition(all 0.2s ease);` seems unneeded (not to mention heavy-handed?)